new-website/README.md

```diff
@@ -31,31 +31,51 @@ Website for DeepChem - https://deepchem.io.
 
 ## Model Data Fetching
-- The model data is fetched from the offical [Deepchem repo](https://github.com/deepchem/deepchem/tree/master/docs/source/api_reference) using the `get_models.py` script in the `/utils/models` directory.
+- The model data is fetched from the official [Deepchem repo](https://github.com/deepchem/deepchem/tree/master/docs/source/api_reference) using the `get_models.py` script in the `/utils/models` directory.
 - The script downloads the raw csv files hosted at Github and extracts the necessary data.
 - The script reads the files using pandas, encodes and saves it to json format in the `/deepchem/data/models` directory.
 - The data is generated during the workflow process and not stored in the repository.
 
+## Tutorial Data Fetching
+- The tutorial data is fetched from the official [Deepchem repo](https://github.com/deepchem/deepchem/tree/master/docs/source/api_reference) using the `fetch_tutorials.py` script in the `/utils/tutorials` directory.
+- The script downloads the raw `.ipynb` files from Github and uses `jupyter nbconvert` to convert them to `html` files.
+- A second script `export_tutorials.py` in the same directory reads the html files, creates a react component and a json data-file out of the parsed HTML data and exports them to the `/deepchem/pages/tutorials` and `/deepchem/data/tutorials` directories respectively.
+- The data is generated only during the workflow process and is not stored in the repository.
+
+A detailed description of the working of the scripts is given below. 
+
+- ### `fetch_tutorials.py`
+  - The script first fetches the list of all tutorials present in the deepchem repo. This is done using web scraping. The list is fetched from [deepchem/examples/tutorials](https://github.com/deepchem/deepchem/tree/master/examples/tutorials) and the `assets` and the `README` are ignored
+  - Once the list of tutorials has been fetched, the script now fetches each notebook, uses `jq` to remove the metadata from the downloaded notebook (this is done to prevent errors during conversion), and finally uses `nbconvert` to convert the notebook to an HTML file.
+  - The list of converted notebooks is stored in `/utils/tutorials/notebooks.txt` which is used by the `export_tutorials.py` script to generate the react components and the json data files.
+
+- ### `export_tutorials.py`
+  - This script reads the list of notebooks from `/utils/tutorials/notebooks.txt` and parses the HTML files using `BeautifulSoup`.
+  - The script then creates a react component for each tutorial and exports it to the `/deepchem/pages/tutorials` directory.
+  - The script also creates a json data file for each tutorial and exports it to the `/deepchem/data/tutorials` directory.
+  - The template for the react components is stored in `utils/tutorials/tutorial_component_template.py`.
+Please note, that any files required by scripts are generated by the scripts themselves and are not stored in the repository.
```

Comment:
Can you add a section about how to run the unit tests for the tutorial fetching?